### PR TITLE
Create README.md

### LINKED ISSUE #20: Add README
Include:
* Build status, i.e. CI badges
* Short description
* Example

### PR DESCRIPTION
**Pull request by andris-zalitis**
Close #20

### CODE REVIEW

**Comment by rudolf-adamkovic** [comment]
@andris-zalitis I removed me and @ruslankmlv from _Reviewers_ and added @enfluid as he is a native English speaker.

**Review by rudolf-adamkovic — DISMISSED** [review]
The commit is:

> andris-zalitis committed with andris-zalitis 

Needs to be fixed. Also it should auto-close #20.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Build status:
+
+| master | develop |
+|--------|---------|
+| [![BuddyBuild](https://dashboard.buddybuild.com/api/statusImage?appID=585be85405e37f01000a9a72&branch=master&build=latest)](https://dashboard.buddybuild.com/apps/585be85405e37f01000a9a72/build/latest?branch=master) | [![BuddyBuild](https://dashboard.buddybuild.com/api/statusImage?appID=585be85405e37f01000a9a72&branch=develop&build=latest)](https://dashboard.buddybuild.com/apps/585be85405e37f01000a9a72/build/latest?branch=develop) |
+
+# About
+
+_Scrub Bar_ is a `UIControl` inspired by Emoji bar in iOS keyboard. It enables tapping and swiping (_scrubbing_) between Scrub Bar items. Each item is represented by an icon and a label for accessibility.
```
Comment (enfluid) [inline README.md]: I'd re-word this to say, "inspired by the Emoji bar in iOS's keyboard"

**Review by enfluid — COMMENTED** [review]
Just added one small recommendation.  Otherwise, looks great!

**Comment by andris-zalitis** [comment]
I updated wording and fixed the issue with commit's author name

**Comment by rudolf-adamkovic** [comment]
@andris-zalitis 

> I updated wording and fixed the issue with commit's author name

Does this mean the PR is ready for review again? If so, please use the _Reviewers_ field.

**Review by enfluid — APPROVED** [review]
LGTM!

**Comment by enfluid** [comment]
@salutis Looks like you have to redo your review as well before it can be merged.

**Comment by andris-zalitis** [comment]
@salutis 
> Does this mean the PR is ready for review again? If so, please use the Reviewers field.

This explains what I have changed. I did use Reviewers section to request a review from @enfluid.

**Pull request merged**